sahi/predict.py

```diff
@@ -460,14 +473,38 @@ def predict(
     # iterate over source images
     durations_in_seconds["prediction"] = 0
     durations_in_seconds["slice"] = 0
-    for ind, image_path in enumerate(tqdm(image_path_list, "Performing inference on images")):
+
+    # Input video
+    if input_video and not export_visual:
+
+        read_video_frame, image_base = get_video_reader(source, save_dir, export_visual, view_image, fast_forwarding)
+        item = read_video_frame
+
+    elif input_video and export_visual:
+
+        read_video_frame, out, image_base = get_video_reader(
+            source, save_dir, export_visual, view_image, fast_forwarding
+        )
+        item = read_video_frame
```

Comment:
this function should be called as `read_video_frame()`, i believe

Reply:
We make `read_video_frame()` function that return of `get_video_reader` assign to `read_video_frame` variable. Therefore, we don't use `read_video_frame()`.